src/pybind/mgr/nfs/cluster.py

```diff
@@ -105,7 +106,11 @@ def create_nfs_cluster(
             ingress: Optional[bool] = None,
             port: Optional[int] = None,
     ) -> Tuple[int, str, str]:
+
         try:
+            if virtual_ip:
+                ip = virtual_ip.split('/')[0]
+                ipaddress.ip_address(ip)
```

Comment:
I assume that this relies on the ip_address function raising an exception. That's good, but maybe a short comment on stuff like this in the future helps readers understand that you're relying on the exception and  the return value of the function is being discarded on purpose. But it's not a big deal. :-)

Reply:
yeah, your are right, I'll add a comment to clarify what is the code is trying to validate :)